server/models/appt.model.ts

```diff
@@ -0,0 +1,36 @@
+import { Document, Model, model, Schema } from "mongoose";
+
+export interface IAppointment extends Document {
```

Comment:
Do we need to extend Document? I see in the Mongoose docs that it's common, but they don't recommend it. Seems if we just line things up properly, there should be no issues. See here: https://mongoosejs.com/docs/typescript.html#using-extends-document

Reply:
I was working off the boilerplate code.  It can be edited.